### PR TITLE
Add debug command to get object paths for pools, filesystems, blockdevs

### PR DESCRIPTION
**Pull request by mulkieran**
Closes https://github.com/stratis-storage/stratis-cli/issues/864

### CODE REVIEW

**Comment by mulkieran** [comment]
Expect coverage to fail

**Comment by mulkieran** [comment]
@drckeefe How do you like this interface ?

**Review by drckeefe — APPROVED** [review]

**Comment by mulkieran** [comment]
@drckeefe I've added similar for blockdev and filesystem. Please take a look. If you approve that, then I'll add man pages.

**Review by drckeefe — APPROVED** [review]

**Comment by mulkieran** [comment]
@drckeefe, @jbaublitz Any requests? Otherwise, I'll squash and merge.

**Review by jbaublitz — APPROVED** [review]
Looks good to me!

File: src/stratis_cli/_parser/_physical.py [hunk 1]
```diff
@@ -30,5 +31,12 @@
             ],
             func=PhysicalActions.list_devices,
         ),
-    )
+    ),
+    (
+        "debug",
+        dict(
+            help=("Miscellaneous filesystem-level debug commands"),
```
Comment (drckeefe) [inline src/stratis_cli/_parser/_physical.py:38]: blockdev-level debug commands

Comment (mulkieran) [reply src/stratis_cli/_parser/_physical.py]: Fixed.

**Comment by mulkieran** [comment]
squashed...

**Pull request merged**